.github/labeler.yml

```diff
@@ -57,6 +57,7 @@
 - 'src/cpp/src/whisper_pipeline.cpp'
 - 'src/python/py_whisper_pipeline.cpp'
 - 'tests/python_tests/test_whisper_generate_api.py'
```

Comment:
Did we miss to add `test_llm_pipeline_static.py` to `LLM category`?

Reply:
I guess so. Just noticed that whisper static pipeline hpp/cpp files also not here.

Reply:
Added whisper files

Reply:
Thank you!